### PR TITLE
Add biscot

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/biscot/biscot.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <assert_contents>
+                <has_text text="Super-Scaffold_1"/>
+                <has_text text="4753350"/>
+                <has_n_lines n="3"/>
+            </assert_contents>
+        </output>
+    </test>
+    </tests>
+    <help><![CDATA[
+    Hello, world!
```
Comment (bgruening) [inline tools/biscot/biscot.xml:104]: This needs some love and the indentation is not always consistent as it seems.

Comment (gallardoalba) [reply tools/biscot/biscot.xml:104]: Sure, I'm improving the help section, because I forgot about that.

**Comment by gallardoalba** [comment]
It works correctly in local, but not when using Docker, so the problem seems to be due to a conflict with a dependency when using Docker @bgruening

**Comment by gallardoalba** [comment]
It was able to solve the conflicts, but took a long time; I disabled the functionality that is causing the problem because is not required at all for the ERGA pipeline.

**Comment by gallardoalba** [comment]
It is still failing, but I think that by the problem is related to the time required to install the dependencies.

File: tools/biscot/biscot.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            --xmap-2enz '${xmap_2enz}'
+        #end if
+        ## $aggressive # disabled due to dependency conflicts
+        $only_confirmed_pos
+    ]]>    </command>
+    <inputs>
+        <param argument="--cmap-ref" type="data" format="cmap" label="Reference CMAP file" help="This file describes the positions of enzymatic labelling sites on the reference genome" />
+        <param argument="--cmap-1" type="data" format="cmap" label="Query CMAP file" help="This file describes the positions of enzymatic labelling sites on the contigs" />
+        <param argument="--xmap-1" type="data" format="xml,txt" label="XML configuration file" help="This file describes the alignments of contig labels on the anchor" />
+        <param argument="--contigs" type="data" format="fasta" label="Contigs file" help="Contigs file (FASTA) that was scaffolded by the Bionano scaffold" />
```
[the remark this answers is not in the document]
Comment (gallardoalba) [reply tools/biscot/biscot.xml:28]: Not, it is not supported, I just included it compressed in order to save space. Should I uncompress it?

[the remark this answers is not in the document]
Comment (nsoranzo) [reply tools/biscot/biscot.xml:28]: No, I think it's fine, I was just checking.

**Comment by bgruening** [comment]
@gallardoalba does it succeed locally with Docker at some point?

**Comment by gallardoalba** [comment]
> @gallardoalba does it succeed locally with Docker at some point?

Yes, but it takes a long time.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Fixed it! Thanks @gallardoalba

**Comment by gallardoalba** [comment]
> Fixed it! Thanks @gallardoalba

Thanks a lot!

**Pull request merged**